src/main/resources/logback.xml

```diff
@@ -17,6 +17,9 @@
     <!-- ALL TRACE DEBUG INFO WARN ERROR OFF -->
     <logger name="com.spotify.docker" level="WARN" />
     <logger name="org.seleniumhq.jetty9" level="WARN" />
+    
+    <!-- SET TO DEBUG TO GET STATUS LOGGING -->
+    <logger name="de.zalando.ep.zalenium.proxy.AutoStartProxySet.Status" level="INFO" />
```

Comment:
Do we need to set log level for this explicitly? The log level is set to logback.loglevel (defaulted to info) and can be set to debug using --debugEnabled flag to zalenium.

Reply:
It isn't necessary. In the past, I've explicitly (though unnecessarily) set debug levels to make it obvious to others how to turn on specific types of logging.

Happy to remove if that is the consensus :)

Reply:
I wonder when this logging is most useful? For people new to zalenium? Which makes me think that it should be opt-out maybe? So default it to on, but have an environment variable that can turn it off for people who don't want it? Or maybe it's just too much noise to have it on by default?

I don't know 😉

Reply:
When JSON logging is isn't enabled this should be on as default is my opinion